src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue

```diff
@@ -118,21 +98,41 @@
       <v-col sm="12" md="10" lg="8" xl="6">
         <WorkExperienceReferenceList :references="modelValue" @edit="handleEdit" @delete="handleDelete" />
       </v-col>
-      <v-col cols="12" class="mt-6">
-        <v-row justify="start" class="ml-1">
-          <v-btn prepend-icon="mdi-plus" rounded="lg" color="alternate" :disabled="isDisabled" @click="handleAddReference">Add References</v-btn>
-        </v-row>
-      </v-col>
+      <v-row class="my-6">
+        <v-col sm="12" md="10" lg="8" xl="6">
+          <Callout v-if="count >= 6 && totalHours < hoursRequired" title="Max limit reached" type="warning" class="mt-10">
+            You reached the limit of 6 work experience references but do not meet the 500 hour requirement. You can still proceed to submit your application.
```

Comment:
small one, 500 hour requirement. We can use hoursRequired. This would accommodate the 400 hour flow too.

Reply:
Good catch